src/cmap/wire_protocol/shared.ts

```diff
@@ -41,6 +41,7 @@ export function applyCommonQueryOptions(
 ): CommandOptions {
```

Comment:
this function is unused in the driver, no need to modify it

Reply:
If it's unused, should I just go ahead and remove the function entirely then?

Reply:
Not in this PR.  But go ahead and open a follow up PR to remove it (and ticket) 🙂 

I'm very particular about atomic commits in our history.